fhttp/httprunner.go

```diff
@@ -187,10 +193,11 @@ func RunHTTPTest(o *HTTPRunnerOptions) (*HTTPRunnerResults, error) {
 	for i := 0; i < numThreads; i++ {
 		// Get the report on the IP address each thread use to send traffic
 		host, ip := httpstate[i].client.GetIPAddress()
-		log.Infof("[%d] %s resolved to %s\n", i, host, ip)
+		currentSocketUsed := httpstate[i].client.Close()
+		log.Infof("[%d] %d socket used, %s resolved to %s\n", i, currentSocketUsed, host, ip)
```

Comment:
trying to remember why we have the `host` there, can it be somehow different for 1 connection than another? (I don't think so but there must be some reason we put it in the interface for GetIPAddress()?)

Reply:
updated